pod/perldiag.pod

```diff
@@ -7972,7 +7972,8 @@ and their use is legal.
 =item Use of uninitialized value%s
 
 (W uninitialized) An undefined value was used as if it were already
-defined.  It was interpreted as a "" or a 0, but maybe it was a mistake.
+defined.  It was interpreted as a "" or a 0, but it was a mistake for
```

Comment:
"but it was a mistake for Perl to do so" is much stronger than "but maybe it was a mistake". I would argue it is too strong.